zhaquirks/tuya/ts0603_garage.py

```diff
@@ -0,0 +1,19 @@
+"""Tuya garage door opener."""
+
+from zigpy.quirks.v2.homeassistant.binary_sensor import BinarySensorDeviceClass
+
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+
+(
+    TuyaQuirkBuilder("_TZE608_fmemczv1", "TS0603")
+    .tuya_onoff(dp_id=1)
```

Comment:
Would a button be more appropriate here?

Reply:
Probably :smile:  This is my first quirk so it wasn't easy. I've searched the repo and there aren't many examples of exposing entity as button in Homeassistant. Finally I "invented" something working and hopefully it is acceptable :smile: